README.md

```diff
@@ -1,4 +1,4 @@
-## Cognite function template
+## Cognite function template [1.3.0]
```

Comment:
Added this template version here as well - I think that was your idea - so that we can quickly know what version of the template was cloned when helping out with projects.

Reply:
Great idea